packages/sourcemap-upload/README.md

```diff
@@ -28,50 +28,68 @@ and vice-versa.
 This module exports a named `uploadSourceMaps` function with the following options:
 
 ```typescript
-interface Options {
+export interface UploadOptions {
   /**
    * The files/directories to search for sourcemaps. All files that match the
    * 'extensions' list and fail to match 'ignore' will be searched for
    * sourcemap JSON or `//#sourceMappingURL=` coments in order to find pairs
    * of generated-file + sourcemap, and the sourcemap will be uploaded.
    */
   filepaths: Array<string> | string;
-
   /**
    * To allow for tracking and browsing of maps that have been uploaded, we
    * require uploaded sourcemaps to have an overall group name associated with
    * them. This could for instance be a version number, or commit hash.
    */
   group: string;
-
   /**
    * The API key to use when connecting to Replay's servers.
    * Defaults to `process.env.REPLAY_API_KEY`.
    */
   key?: string;
-
   /**
    * Run all of the local processing and searching for maps, but skip uploading them.
    */
   dryRun?: boolean;
-
+  /**
+   * Delete all found sourcemap files after they have been uploaded.
+   */
+  deleteAfterUpload?: boolean;
+  /**
+   * If sourcemaps can't be matched to generated files by their sourceMappingURL, try matching by filenames on disk
+   */
+  matchSourcemapsByFilename?: boolean;
```

Comment:
Is there any reason not to default this to `true`? It seems like very reasonable default behavior.

Reply:
Can't immediately think of a reason _against_ having a default of `true`.

In practice, this shouldn't matter often, because normally generated bundles _will_ have a `sourceMappingURL` comment.  I just happened to find that they _don't_ when you set `devtool: "hidden-source-map"`, and only tried that because the delete plugin suggested it to avoid 404s.

Reply:
> I just happened to find that they don't when you set devtool: "hidden-source-map"

Isn't this a popular setting though? At least this is what I was always using to avoid my source maps being publicly available.

I'd probably make this a default too if we don't have any concerns around this.

Reply:
I'd actually never even _heard_ of `"hidden-source-map"` until now :)

But yes, went ahead and made this setting default to `true`